solver/src/metrics.rs

```diff
@@ -104,4 +129,21 @@ impl SolverMetrics for NoopMetrics {
     fn liquidity_fetched(&self, _liquidity: &[Liquidity]) {}
     fn settlement_computed(&self, _solver_type: &str, _start: Instant) {}
     fn order_settled(&self, _order: &Order) {}
+    fn settlement_simulation_successes(&self, _count: usize) {}
```

Comment:
naming nit: to be inline with the other methods maybe this can be called settlement_simulation_succeeded/failed

Reply:
changed